pennylane_forest/ops.py

```diff
@@ -24,75 +24,16 @@
 Code details
 ~~~~~~~~~~~~
 """
+import pennylane as qml
 from pennylane.operation import Operation
 
 
-class S(Operation):
-    r"""S(wires)
-    S gate.
-
-    .. math:: S = \begin{bmatrix} 1 & 0 \\ 0 & i \end{bmatrix}
-
-    **Details:**
-
-    * Number of wires: 1
-    * Number of parameters: 0
-
-    Args:
-        wires (int): the subsystem the gate acts on
-    """
-    num_params = 0
-    num_wires = 1
-    par_domain = None
-
-
-class T(Operation):
-    r"""T(wires)
-    T gate.
-
-    .. math:: T = \begin{bmatrix}1&0\\0&e^{i \pi / 4}\end{bmatrix}
-
-    **Details:**
-
-    * Number of wires: 1
-    * Number of parameters: 0
-
-    Args:
-        wires (int): the subsystem the gate acts on
-    """
-    num_params = 0
-    num_wires = 1
-    par_domain = None
-
-
-class CCNOT(Operation):
-    r"""CCNOT(wires)
-    Controlled-controlled-not gate.
-
-    .. math::
-
-        CCNOT = \begin{bmatrix}
-            1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
-            0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\
-            0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\
-            0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\
-            0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 \\
-            0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 \\
-            0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 \\
-            0 & 0 & 0 & 0 & 0 & 0 & 1 & 0
-        \end{bmatrix}
-
-    **Details:**
-
-    * Number of wires: 3
-    * Number of parameters: 0
-
-    Args:
-        wires (int): the subsystem the gate acts on
-    """
-    num_params = 0
-    num_wires = 3
-    par_domain = None
+# We keep the following definitions for compatibility
+# as they are now part of PennyLane core
+S = qml.S
+T = qml.T
+CSWAP = qml.CSWAP
+CCNOT = qml.Toffoli
```

Comment:
Yeah, but there is no immediate need to do so. But depreceation messages are a good idea!

Reply:
How would you add the depreceation message here?

Reply:
Not very well - it might require some weird functional/class based approach to mimic a variable (the same way we added a deprecation message to the entire `qml.expval.` module). Less effort to just delete I think :laughing:

Reply:
Then I'd just leave it like it is for now and we delete it after there is a new release of PL-F?